services/postgres/docker-entrypoint-initdb.d/create-readonly-user.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e
+
+#
+# This script are ONLY run if you start the container with a data directory that is EMPTY (i.e. first time)
```

Comment:
so this will only work on non AWS deployments. Maybe the migration service would have been a better place for this.

Reply:
I see your point. The problem is that this is a one-time-thing. I thought it should rather be in the building scripts from ops

Reply:
I agree with @GitHK. It looks like simcore repository is not the right place to have this logic.